### PR TITLE
[X11] Make children windows to dissappear once user presses outside t…

### PR DESCRIPTION
**Pull request by msisov**
…he main window

This patch starts to use GrabPointer method, which starts to report all the events, which are
happen even outside the main window. Thus, it fixes, for example, hiding the 3-dot menu, when user
presses "New Window" in that one.

### CODE REVIEW

File: ui/base/x/x11_pointer_grab.cc [hunk 1]
```diff
@@ -43,9 +44,9 @@ int GrabPointer(XID window, bool owner_events, ::Cursor cursor) {
         ui::DeviceDataManagerX11::GetInstance()->master_pointers();
     for (int master_pointer : master_pointers) {
       evmask.deviceid = master_pointer;
-      result = XIGrabDevice(
-          gfx::GetXDisplay(), master_pointer, window, CurrentTime, cursor,
-          GrabModeAsync, GrabModeAsync, owner_events, &evmask);
+      result = XIGrabDevice(gfx::GetXDisplay(), master_pointer, window,
```
Comment (tonikitoo) [inline ui/base/x/x11_pointer_grab.cc]: if it is just a style fix from 'git cl format' I would leave it alone.

File: ui/base/x/x11_pointer_grab.h [hunk 2]
```diff
@@ -0,0 +1,29 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
```
Comment (tonikitoo) [inline ui/base/x/x11_pointer_grab.h:1]: 2017?

**Review by tonikitoo — APPROVED** [review]
lgtm. good upstreaming candidate too.

File: ui/base/x/x11_pointer_grab.h (hunk 2 above)
Comment (msisov) [reply ui/base/x/x11_pointer_grab.h:1]: I'm not sure. I guess when one moves the file around, the year mustn't be changed. Only if the file is completely new.

**Comment by msisov** [comment]
I'll make an issue and mention to upstream it

**Pull request merged**